make/Init.gmk

```diff
@@ -160,7 +160,8 @@ main: $(INIT_TARGETS)
 	            -f make/Main.gmk $(USER_MAKE_VARS) \
 	            $(PARALLEL_TARGETS) $(COMPARE_BUILD_MAKE) $(BUILD_LOG_PIPE) || \
 	        ( exitcode=$$? && \
-	        $(PRINTF) "\nERROR: Build failed for $(TARGET_DESCRIPTION) (exit code $$exitcode) \n" \
+	        $(ECHO) "" $(BUILD_LOG_PIPE_SIMPLE) && \
```

Comment:
Just for my understanding, is the empty string needed, or "just" a stylistic choice?

Reply:
Yeah, it is a stylistic choice to indicate that the line was intentionally left empty.